converter/internal/staticconvert/testdata/promtail_scrape.diags

```diff
@@ -0,0 +1,2 @@
+(Error) global positions configuration is not supported - each Flow Mode's loki.source.file component has its own positions file in the component's data directory
```

Comment:
These concern me that every single static promtail config will have this error and warning currently. I'm not sure what we can do about it.

Reply:
Yeah... it could be a warning, but I do want it to catch attention. It's kinda like with the CLI arguments. Losing this (the CLI args or specific positions file location) may be harmless for many, but probably there is a good 20+% of cases out there for which it could be breaking the setup.